test/routes.test.js

```diff
@@ -123,3 +283,39 @@ testWithServer('Request for Person JSON Page', (t, ctx) => {
     t.end();
   });
 });
+
+testWithServer('Request for Person JSON Page for a wrong id', (t, ctx) => {
+  t.plan(4);
+
+  const htmlRequest = {
+    method: 'GET',
+    url: '/people/smgc-people-wrongid',
+    headers: {'Accept': 'application/vnd.api+json'}
+  };
+
+  ctx.server.inject(htmlRequest, (res) => {
+    var response = JSON.parse(res.payload);
+    t.equal(res.statusCode, 200, 'Status code was as expected');
+    t.ok(response.status, 404, 'status is 404');
```

Comment:
So the HTTP statusCode is 200 but the _response_ status is 404 ...? ("_successfully not found_") 😉  ?